src/core/search/rax_tree.h

```diff
@@ -92,6 +93,11 @@ template <typename V> struct RaxTreeMap {
   }
 
   ~RaxTreeMap() {
+    for (auto it = begin(); it != end(); ++it) {
+      V* ptr = &(*it).second;
+      std::allocator_traits<decltype(alloc_)>::destroy(alloc_, ptr);
+      alloc_.deallocate(ptr, 1);
+    }
```

Comment:
I really forgot this 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻 🤦🏻

Reply:
<img src="https://github.com/user-attachments/assets/4d682edd-c156-4094-b50f-619dc1379e9a" width=400/>